spotinst_sdk2/clients/hpc/__init__.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from spotinst_sdk2.client import Client
+
+# region AWS imports
+import spotinst_sdk2.models.hpc.aws as aws_hpc
+# endregion
+
+
+# region AWS
+
+
+class HPCAwsClient(Client):
+    __base_hpc_url = "/hpc/aws/lsf/cluster"
+    __base_aws_url = "/hpc/aws/lsf"
+
+ # endregion
+
+    # region HPCCluster
+    def create_hpc_cluster(self, cluster:aws_hpc.HPC, async_scale=None):
```

Comment:
async_scale parameter not applicable to this cluster. Remove it.

Reply:
removed